bom-latest/pom.xml

```
@@ -11,7 +11,7 @@
     <properties>
         <checks-api.version>1.2.0</checks-api.version>
         <configuration-as-code-plugin.version>1.46</configuration-as-code-plugin.version>
-        <git-plugin.version>4.5.0</git-plugin.version>
+        <git-plugin.version>4.5.1</git-plugin.version>
```

Try git plugin 4.5.2 to confirm it resolves the issue.  I don't know why the `USER` environment variable in the CI environment for this job has an empty string as its value, but the tests should now handle that case.
```suggestion
        <git-plugin.version>4.5.2</git-plugin.version>
```